flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -1115,6 +1115,12 @@ public void run() {
 						checkpointMetaData.getCheckpointId());
 				}
 			} catch (Exception e) {
+				if (LOG.isDebugEnabled()) {
+					LOG.debug("{} - asynchronous part of checkpoint {} could not be completed.",
```

Comment:
IMO, it could be an INFO level Log.

Reply:
First, I want to use WARN or even ERROR level, but the frequency of checkpoint may be too small, we may have logging flood if use INFO or above log level, so I used debug here. If we do not have so frequent checkpoints, we can configure the log level of log-4j properties to show this log. what do you think about this?

Reply:
Make sense. +1 for merge.